packages/wrangler/src/utils/deployment-id-version-id-change.ts

```diff
@@ -0,0 +1,12 @@
+import { logger } from "../logger";
+
+export function logVersionIdChange() {
+	const docsLink =
+		"https://developers.cloudflare.com/workers/configuration/versions-and-deployments";
+
+	logger.log("");
+	logger.log("");
```

Comment:
Why not just add some `\n` to the start of the log below?

Reply:
I first logged the message with logger.warn and the prefix WARNING was added before the "\n\n"

Reply:
https://github.com/cloudflare/workers-sdk/pull/5400/commits/64ed9086802df55c13e6862510f2881fbc25ed2a